src/main/java/net/mcreator/ui/laf/renderer/elementlist/TilesModListRender.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2020 Pylo and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.laf.renderer.elementlist;
+
+import net.mcreator.minecraft.MCItem;
+import net.mcreator.ui.init.TiledImageCache;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.ui.laf.MCreatorTheme;
+import net.mcreator.util.StringUtils;
+import net.mcreator.util.image.ImageUtils;
+import net.mcreator.workspace.elements.FolderElement;
+import net.mcreator.workspace.elements.IElement;
+import net.mcreator.workspace.elements.ModElement;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class TilesModListRender extends JPanel implements ListCellRenderer<IElement> {
+
+	private final boolean showText;
+
+	public TilesModListRender(boolean showText) {
```

Comment:
You said you removed unused code, but what is showText doing here then?

Reply:
you want me remove showText in all renders or only this?